src/constants/api-payloads/layers.ts

```diff
@@ -0,0 +1,144 @@
+export const LAYERS = [
```

Comment:
for mocking purposes my recommendation is working with JSON files with the format provided from the API